app/src/test/java/com/example/sharingang/items/InMemoryItemRepositoryTest.kt

```diff
@@ -0,0 +1,43 @@
+package com.example.sharingang.items
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import com.example.sharingang.getOrAwaitValue
+import kotlinx.coroutines.runBlocking
+import org.junit.Rule
+import org.junit.Test
+
+class InMemoryItemRepositoryTest {
+    @get:Rule
+    val rule = InstantTaskExecutorRule()
+
+    @Test
+    fun inMemoryItemRepositoryStartsEmpty() {
+        val repo: ItemRepository = InMemoryItemRepository()
+        runBlocking {
+            assert(repo.getAllItems().isEmpty())
+        }
+    }
+
+    @Test
+    fun inMemoryItemRepositoryCanRetrieveStoredData() {
```

Comment:
it might be a good idea to split this test up?

The description is a bit vague, and there's quite a few different assertions.

Reply:
Yes, I'll split it.